Docs/ProjectSpec.md

```diff
@@ -2,44 +2,52 @@
 
 ### Index
 
-- [General](#general)
-- [Project](#project)
-     - [Include](#include)
-     - [Options](#options)
-     - [GroupOrdering](#groupordering)
-     - [FileType](#filetype)
-     - [Configs](#configs)
-     - [Setting Groups](#setting-groups)
-- [Settings](#settings)
-- [Target](#target)
-     - [Product Type](#product-type)
-     - [Platform](#platform)
-     - [Sources](#sources)
-     - [Dependency](#dependency)
-     - [Config Files](#config-files)
-     - [Plist](#plist)
-     - [Build Script](#build-script)
-     - [Build Rule](#build-rule)
-     - [Target Scheme](#target-scheme)
-     - [Legacy Target](#legacy-target)
-- [Aggregate Target](#aggregate-target)
-- [Target Template](#target-template)
-- [Scheme](#scheme)
-     - [Build](#build)
-     - [Common Build Action options](#common-build-action-options)
-     - [Execution Action](#execution-action)
-     - [Run Action](#run-action)
-     - [Test Action](#test-action)
-     - [Archive Action](#archive-action)
-     - [Simulate Location](#simulate-location)
-     - [Scheme Management](#scheme-management)
-     - [Environment Variable](#environment-variable)
-     - [Test Plan](#test-plan)
-- [Scheme Template](#scheme-template)
-     - [Remote Package](#remote-package)
-     - [Local Package](#local-package)
-- [Swift Package](#swift-package)
-- [Project Reference](#project-reference)
+- [Project Spec](#project-spec)
```

Comment:
Not sure we need this single header and then everything indented beneath. I can fix in master